cmd/oci-image-validate/main.go

```diff
@@ -89,48 +81,58 @@ func newValidateCmd(stdout, stderr *log.Logger) *cobra.Command {
 		`Print version information and exit`,
 	)
 
+	cmd.Flags().StringVar(
+		&v.logLevelString, "log-level", "error",
+		`Log level (panic, fatal, error, warn, info, or debug)`,
+	)
+
 	return cmd
 }
 
 func (v *validateCmd) Run(cmd *cobra.Command, args []string) {
+	logLevel, err := logrus.ParseLevel(v.logLevelString)
+	if err != nil {
+		logrus.Fatal(err)
+	}
+	logrus.SetLevel(logLevel)
+
 	if v.version {
-		v.stdout.Printf("commit: %s", gitCommit)
-		v.stdout.Printf("spec: %s", specs.Version)
+		fmt.Printf("commit: %s\n", gitCommit)
+		fmt.Printf("spec: %s\n", specs.Version)
 		os.Exit(0)
 	}
 
 	if len(args) < 1 {
-		v.stderr.Printf("no files specified")
 		if err := cmd.Usage(); err != nil {
-			v.stderr.Println(err)
+			logrus.Errorf("%s", err)
```

Comment:
fixed in https://github.com/opencontainers/image-tools/pull/46/commits/9ae60ce0c8a7131f0eb5e03316541f517ae93aa1.
To use logrus.WithError(err).Errorf() to instead logrus.Errorf(), on related positions.